packages/medusa/src/services/pricing.ts

```diff
@@ -268,114 +282,128 @@ class PricingService extends TransactionBaseService {
         .withTransaction(this.activeManager_)
         .retrieve(variantId, { select: ["id", "product_id"] })
 
-      productRates = await this.taxProviderService
-        .withTransaction(this.activeManager_)
-        .getRegionRatesForProduct(product_id, {
-          id: pricingContext.price_selection.region_id,
-          tax_rate: pricingContext.tax_rate,
-        })
+      productRates = (
```

Comment:
thought(this-file, non-blocking): This goes along with my other comment. I generally think we should be explicit about functions that return maps i.e. define a const with a name that reflects it. And then subsequently use the `.get( ... )` to fetch what we want. Wdyt?